RELEASING.md

```diff
@@ -7,9 +7,9 @@ Setup:
 Then:
 
 1. Create a new branch: `release/vX.Y.Z`
-2. Update the versions all `Cargo.toml` files, as necessary
+2. Update the versions of all `Cargo.toml` files to match
```

Comment:
What do you think about putting `version` in the workspace `Cargo.toml` so that we don't have to update all the files manually?

Reply:
https://github.com/kylebarron/arro3/blob/2d9861ff2c75453f0b65081bdd5b42d28d2d5e13/Cargo.toml#L6-L15

https://github.com/kylebarron/arro3/blob/2d9861ff2c75453f0b65081bdd5b42d28d2d5e13/arro3-core/Cargo.toml#L2-L13

Reply:
Ah nice, I like that. I google-failed on whether I could do that. Will fix.